src/Apps/Settings/Routes/EditSettings/Components/SettingsEditSettingsInformation.tsx

```diff
@@ -155,24 +176,23 @@ export const SettingsEditSettingsInformation: React.FC<SettingsEditSettingsInfor
                 title="Mobile Number"
                 mt={4}
                 inputProps={{
-                  name: "phone",
+                  name: "phoneNumber",
```

Comment:
I think this change is related to [Integrity failure](https://circleci-tasks-prod.s3.us-east-1.amazonaws.com/storage/artifacts/0c063670-0ce1-4c4b-a09a-d12b1e37d38b/769f615b-dae2-4d78-8b34-9141ebe5ddea/0/cypress/screenshots/editSettings.cy.ts/settingsedit-settings%20--%20can%20update%20basic%20information%20%28failed%29%20%28attempt%202%29.png?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=ASIAQVFQINEOMHPWNV4Z%2F20231006%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20231006T111137Z&X-Amz-Expires=60&X-Amz-Security-Token=IQoJb3JpZ2luX2VjEGsaCXVzLWVhc3QtMSJHMEUCIQCj1hBgRtW9032yAigRMjrdZ0DRbWmbVBdgvS1PBYTaOgIgGg77i6Q3Mr1DGhqcJDzQb%2FcjRGJLa82g7oAmTDcB8IYqqwIIdBADGgwwNDU0NjY4MDY1NTYiDHwphWJ0U3DVVOMsGyqIAjvHMFU8HaCd2nm20LJVmsILET%2BYVD2YAjEOTbJ5QCvEQhy0LtU5LtfaRiu0nHSDOkTP6DknhQGfzPJ199s0CbyXt9CXW2VBA2p340TyZiY4pa4wBKIj6su1gHyO8V3rsezn2Tj%2B0x2a%2BJnu9tUh%2FtvQtcw0qAUgtrKy320ktRaPlJeBf2zRU3j0zoFquie3o15YYIq1YKmF%2FJrSLe8kppLcey9VBErARQgtjhpbe3dkEzRyefCIdzPU%2B1pQyXwPrMQwuiam33Zgt8oxOhGHIbe1VQrUkGoma24UFOm83n0zvXP7yYlGZJpWhZNztq3PW%2FwTdEgVgOO13O9pkUnTEF0OK1uQg2o13zCA0%2F%2BoBjqdARXg%2BEuX86NH2ndfE%2BurvmBpBLZhbrsBXhVErHRXJ8hZ%2B%2FdtCl7uiDHmt1N3gWymJerZUcVmvWxE9dwM7GVHRyuaKIqnYF5yNfxwByySl0vnOWEkiCObvQ4SYJXscT8BAezJOyNNICV16dJ9DhqOFC7cnDBCdRbVJ7Fo93Q3NEOa5WTxrGrlkCd0HrV3sYL865QwCb1met%2BGJjoQVoY%3D&X-Amz-SignedHeaders=host&x-id=GetObject&X-Amz-Signature=c7f422305fb9ab8d86a1c758668b971decf34bbac3c958bbfc685572778a3562)

Reply:
Yes, thanks for the integrity PR! 💜

Reply:
You are welcome :)